src/raft.rs

```diff
@@ -268,14 +260,13 @@ impl<T: Storage> Raft<T> {
             tag: c.tag.to_owned(),
         };
         for p in peers {
-            let pr = new_progress(1, r.max_inflight);
+            let pr = Progress::new(r.raft_log.last_index(), r.max_inflight, false);
```

Comment:
Why change the index?

Reply:
Oh you're right. I'm sorry. :disappointed: